webapp/lib/controller.dart

```diff
@@ -117,6 +164,10 @@ void command(UIAction action, Data actionData) {
     /*** Data */
     case UIAction.projectSelected:
       view.contentView.populateUrlFilters();
+      var selectedProjectTimer = projectTimers[view.contentView.projectSelectorView.selectedProject];
+      if (selectedProjectTimer != null && !selectedProjectTimer.isActive) {
+        view.contentView.stale = true;
+      }
```

Comment:
we need to be able to catch stale projects that have been stale before we loaded nook-watcher (for example: load nook watcher, current selected project is active, switch to another project which hasn't been updated in a day, it should show up as stale). I don't think this catches this? potentially for cases like this, the timer should never be set because it would be set in the past so maybe we can rely on the timer being null to also signal stale projects?

Reply:
If we go with this then isn't the previous comment about creating timers for all non-selected project nullified?